web/client/selectors/__tests__/annotations-test.js

```diff
@@ -433,6 +434,10 @@ describe('Test annotations selectors', () => {
         const retVal = removingSelector(state);
         expect(retVal).toBe(null);
     });
+    it('test multiGeometrySelector', () => {
+        const multiGeometry = multiGeometrySelector(state);
+        expect(multiGeometry).toBeTruthy();
```

Comment:
please test default behaviour

Reply:
Sorry, What's the default behaviour in this context? @MV88

Reply:
multiGeometry to be true